packages/data/src/registry.js

```diff
@@ -102,47 +101,6 @@ export function createRegistry( storeConfigs = {}, parent = null ) {
 		return result;
 	}
 
-	const getResolveSelectors = memize(
```

Comment:
Does removing this usage of `memize` impact performance?

Reply:
I ran `npm run test-performance` with and without this branch and didn't see any regression.

Reply:
Good point. Removing `memize` impacts performance, but I don't think it's measurable in practice.

Without `memize`, the selectors are mapped (i.e., the object with resolution functions is created) from scratch every time `registry.resolveSelect()` is called. This is not needed, the mapping can be done only once.

I'm fixing this in #28544 by doing the mapping only once at store initialization.